lib/sites/infibeam.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const siteUtils = require('../site-utils');
+const logger = require('../logger')();
+
+class InfibeamSite {
+  constructor(uri) {
+    if (!InfibeamSite.isSite(uri)) {
+      throw new Error('invalid uri for Infibeam: %s', uri);
+    }
+
+    this._uri = uri;
+  }
+
+  getURIForPageData() {
+    return this._uri;
+  }
+
+  isJSON() {
+    return false;
+  }
+
+  findPriceOnPage($) {
+    // the various ways we can find the price
+    const selectors = [
+      '#price-after-discount',
+    ];
+
+    // find the price on the page
+    let priceStorage = siteUtils.findContentOnPage($, selectors);
+    if (priceStorage !== null) {
+      priceStorage = priceStorage.substring(2, priceStorage.length);
+      priceStorage = priceStorage.replace(',', '');
+    }
```

Comment:
I think I understand what you're doing here, but think it might be best if we move this logic to the `siteUtils.processPrice` function.  Let me explain:

It looks like when you find the content on the page, you'll get something like this back:
```
R 9,444
```
When you perform the `substring` and the `replace`, you're turning `R 9,444` into `9444`.  This causes the `siteUtils.processPrice` call below to fail to return the price, because it doesn't understand the currency of `9444`.

What I think we should do instead, is remove the `substring` and `replace` calls above, and send the priceString without edit to the `siteUtils.processPrice`.  Then add logic to `processPrice` to support rupees (I'm assuming that's what the R stands for?).  This way `processPrice` will be in charge of converting `R 9,444` to `9444`, and any other sites that use rupees will not have to implement the logic you did above.

Does that make sense?